python/tvm/relay/grammar/py2/RelayLexer.py

```diff
@@ -0,0 +1,209 @@
+# Generated from /home/sslyu/tvm/python/tvm/relay/grammar/Relay.g4 by ANTLR 4.7.2
```

Comment:
Can you mark these all as binary files?

Reply:
I did in .gitattributes, but it didn't change how they display on Github. I will try this as well: https://help.github.com/en/articles/customizing-how-changed-files-appear-on-github